packages/reports/src/Components/Form/CompositeCalendar/composite-calendar.tsx

```diff
@@ -97,36 +113,36 @@ const CompositeCalendar = props => {
         }
     };
 
-    const setToDate = date => {
-        updateState('to', epochToMoment(date).endOf('day').unix());
+    const setToDate = (date: moment.Moment) => {
+        updateState('to', toMoment(date as unknown as number).endOf('day'));
```

Comment:
```suggestion
        updateState('to', toMoment(date).endOf('day'));
```
no need for this type casting as I checked